sourced/ml/core/algorithms/token_parser.py

```diff
@@ -32,15 +32,20 @@ class TokenParser:
     # Example: 'sourced.ml.algorithms' -> ["sourc", "sourcedml", "algorithm", "mlalgorithm"].
     # if True we have only ["sourc", "algorithm"].
     # if you do not want to filter small tokens set min_split_length=1.
-    SAVE_TOKEN_STYLE = False  # whether yield metadata that can be used to reconstruct initial
-    # identifier.
+    SAVE_TOKEN_STYLE = False  # whether yield metadata that can be used to reconstruct
+    # initial identifier
+
+    # default nn model modelforge UUID to load when using neural network splitter
+    NN_MODEL = "522bdd11-d1fa-49dd-9e51-87c529283418"
+    USE_NN = False  # Wether to use or not Neural Network-based splitter
     ATTACH_UPPER = True  # True to attach the last of several uppercase letters in a row to
     # the next token. Example: 'HTMLResponse' -> ["html", "response"] if True,
     # 'HTMLResponse' -> ["htmlr", "esponse"] if False.
 
     def __init__(self, stem_threshold=STEM_THRESHOLD, max_token_length=MAX_TOKEN_LENGTH,
                  min_split_length=MIN_SPLIT_LENGTH, single_shot=DEFAULT_SINGLE_SHOT,
-                 save_token_style=SAVE_TOKEN_STYLE, attach_upper=ATTACH_UPPER):
+                 save_token_style=SAVE_TOKEN_STYLE, attach_upper=ATTACH_UPPER, use_nn=USE_NN,
+                 nn_model=NN_MODEL):
```

Comment:
```suggestion
                 nn_model=None):
```
so that we load the most recent one